py/desitransfer/data/desi_transfer_daemon.ini

```diff
@@ -19,7 +19,7 @@ checksum_file = checksum-{exposure}.sha256sum
 [common]
 # Use this directory for temporary files.  The first available directory
 # in this list will be used.
-scratch = ${CSCRATCH},${HOME}/tmp,${HOME}/scratch
+temporary = ${SCRATCH},${HOME}/tmp,${HOME}/scratch,/tmp
```

Comment:
Don't include `/tmp` here, it's not a good idea to use it on the data transfer nodes.